Double/script.js

```diff
@@ -0,0 +1,89 @@
+const imgData = [
+    {
+      src: "./img/among.png",
+      name: "AmongUs",
+    },
+    {
+      src: "./img/glasses.png",
+      name: "Glasses",
+    },
+    {
+      src: "./img/mango.png",
+      name: "Mango",
+    },
+    ];
+
+const cardBack = "./img/font.png";
+
+const doubleImgArrayCopy =  imgData.concat(imgData);
+const doubleImgArray = doubleImgArrayCopy.map((a, i) => Object.assign({}, a));
+let currentCard = null;
```

Comment:
This var looks useless. You just check and set in on rows 44-45 and refresh on row 83. You do not use this value nowhere